src/sentry/uptime/consumers/results_consumer.py

```diff
@@ -383,6 +405,58 @@ def has_reached_status_threshold(
             )
         return result
 
+    def _produce_snuba_uptime_result(
+        self, project_subscription: ProjectUptimeSubscription, result: CheckResult
+    ) -> None:
+        """
+        Produces a message to Snuba's Kafka topic for uptime check results.
+
+        Args:
+            project_subscription: The project subscription associated with the result
+            result: The check result to be sent to Snuba
+        """
+        try:
+            project = project_subscription.project
+            retention_days = (
+                quotas.backend.get_event_retention(organization=project.organization) or 90
+            )
+
+            snuba_message: SnubaUptimeResult = {
+                # Copy over fields from original result
+                "guid": result["guid"],
+                "subscription_id": result["subscription_id"],
+                "status": result["status"],
+                "status_reason": result["status_reason"],
+                "trace_id": result["trace_id"],
+                "span_id": result["span_id"],
+                "scheduled_check_time_ms": result["scheduled_check_time_ms"],
+                "actual_check_time_ms": result["actual_check_time_ms"],
+                "duration_ms": result["duration_ms"],
+                "request_info": result["request_info"],
+                # Add required Snuba-specific fields
+                "organization_id": project.organization_id,
+                "project_id": project.id,
+                "retention_days": retention_days,
+                "region_slug": result.get("region"),
```

Comment:
since this is just a renaming of `region`, we can rename this to eventually be region and get rid of the name change